src/components/columns/Column.js

```diff
@@ -4,13 +4,14 @@ export default class ColumnComponent extends NestedComponent {
   constructor(component, options, data) {
     super(component, options, data);
     this.noEdit = true;
+    this.hideOnChildrenHidden = this.component.hideOnChildrenHidden != null ? this.component.hideOnChildrenHidden : true;
```

Comment:
We cannot change the default behavior. You are defaulting this to ```true``` when it should be ```false```. Because of this, I think the variable should be called ```showOnChildrenHidden``` and default it to ```false```. 

There is also a better way of doing this where you would just add this new property to the ```schema``` definition. Then you can put this in the check.

```js
if (allChildrenHidden && !this.component.showOnChildrenHidden) {

}
```